remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ObjectRemoteQueryManager.java

```diff
@@ -69,7 +68,7 @@ private ObjectRemoteQueryEngine getQueryEngineForMediaType(MediaType mediaType)
          ReflectionMatcher matcher = mediaType.match(APPLICATION_PROTOSTREAM) ? cr.getComponent(ProtobufObjectReflectionMatcher.class) :
                cr.getComponent(ObjectReflectionMatcher.class);
 
-         queryEngine = new ObjectRemoteQueryEngine(cache, searchIntegrator != null, matcher.getClass());
+         queryEngine = new ObjectRemoteQueryEngine(cache, true, matcher.getClass());
```

Comment:
Shouldn't we pass `false` if no protobuf entity is indexed? I believe there is support for queries even in that case, throught "brute-force" scanning?

Reply:
Again good catch. I don't remember why I plugged here a true. I think I've fixed it, checking the existence of the `indexInspector` in place of the `searchIntegrator`.